examples/shap/README.md

```diff
@@ -0,0 +1,9 @@
+# SHAP Examples
+
+Examples for the SHAP integration (`mlflow.shap`).
+
+| File                                                         | Task                      | Description                                                    |
+| :----------------------------------------------------------- | :------------------------ | :------------------------------------------------------------- |
+| [regression.py](regression.py)                               | Regression                | Log explanations for a LinearRegression model                  |
+| [binary_classification.py](binary_classification.py)         | Binary classification     | Log explanations for a binary RandomForestClassifier model     |
+| [multiclass_classification.py](multiclass_classification.py) | Multiclass classification | Log explanations for a multiclass RandomForestClassifier model |
```

Comment:
Could we explain prerequisites for the example scripts (``pip install mlflow shap`` and  & provide a command for running the scripts? And also describe how to view the resulting runs + artifacts in the MLflow UI?